audit/build.gradle

```diff
@@ -55,9 +55,9 @@ dependencies {
 
     implementation "org.springframework.boot:spring-boot-autoconfigure"
 
-    implementation group: 'org.liquibase', name: 'liquibase-core'
-    implementation group: 'org.liquibase.ext', name: 'liquibase-hibernate5', version: '4.3.3'
-    implementation group: 'com.mattbertolini', name: 'liquibase-slf4j', version: '2.0.0'
+    implementation "org.liquibase:liquibase-core"
+    implementation "org.liquibase.ext:liquibase-hibernate5:4.16.1"
```

Comment:
Can you move version specific stuff to the gradle properties? That way, they can be consistent across all modules.